components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -81,6 +81,48 @@ To access these components for an organization, select **Chef Servers > Organiza
 
 A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuration and policy distribution. A cookbook contains recipes and other files, components, or directories.
 
+The Chef Infra Server UI lets you:
+
+- List out cookbooks with the latest version
+- View different versions of a cookbook
+- View the contents of a cookbook
+- View the details of a cookbook
+
+#### List of Cookbooks
+
+In Chef Infra Server, you can view all the cookbooks of an organization. Each Cookbook has its latest version mentioned in the list.
+
+{{< figure src="/images/automate/list-of-cookbooks.png" alt="List of Cookbooks">}}
+
+Select any cookbook from the list to view the details of that particular cookbook.
+
+#### View Different Versions of Cookbooks
+
+To view the different versions of the cookbook:
+
+- Select a specific cookbook from the list.
+- The interface contains a dropdown list for versions. Select any version to view the details.
+
+{{< note >}}
+The dropdown shows the latest version of the cookbook.
+{{< /note >}}
+
+{{< figure src="/images/automate/view-different-versions-of-cookbook.png" alt="Versions of a Cookbook">}}
+
+#### Cookbook Content
+
+Select the content tab to view the recipes and other optional components of a cookbook. It also contains the files or directories of the components. You can view the content of the file by selecting the file from the component list.
```

Comment:
- Use the indicative `To view` instead of the subjunctive `You can view`
- revise the sentence to use the possessive `to view a cookbook's` instead of the genitive `of a cookbook`
- This simplifies the sentence by moving the sentence's object (`cookbook`) to the second position.